src/axom/primal/tests/primal_curved_polygon.cpp

```diff
@@ -235,18 +236,26 @@ TEST(primal_curvedpolygon, split_edge)
 
   SLIC_INFO("Test checking CurvedPolygon edge split.");
 
-  std::vector<PointType> CP = {PointType {0.6, 1.2},
+  axom::Array<PointType> CP = {PointType {0.6, 1.2},
                                PointType {0.3, 2.0},
                                PointType {0.0, 1.6},
                                PointType {0.6, 1.2}};
 
-  std::vector<int> orders32 = {1, 1, 1};
+  std::cout << CP << std::endl;
+  CP.insert(0, CP[0]);
+  std::cout << CP << std::endl;
+
+  return;
```

Comment:
Is there a reason for the early return?
Please try to remove it and fix any problems that you encounter.

Reply:
... and associated debug code/comments

Reply:
Oops! Leftovers from when I was investigating the buggy behavior with `axom::Arrays`. All cleaned up now.